public/index.html

```diff
@@ -4207,7 +4207,7 @@ <h4 class="title_restorable">
                             <!-- <h4><span data-i18n="UI Colors">Theme Settings</span></h4> -->
                             <div name="AvatarAndChatDisplay" class="flex-container flexFlowColumn">
                                 <div class="flex-container alignItemsBaseline">
-                                    <span data-i18n="Avatar Style">Avatars:</span>
+                                    <span data-i18n="Avatar Style:">Avatars:</span>
```

Comment:
Same here. If you rename it, rename all the keys in existing translations files.